com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -305,6 +280,7 @@ public static void atomicCopy(URL source, File target, boolean executable) throw
 
     private static String getExeWithExtension(Platform platform, String name, String extension) throws IOException {
         init();
+        PackedResources.waitForUnpackLibs();
```

Comment:
I don't really understand the flow and when to do both runUnpackAllLibsAsync() and waitForUnpackLibs() and when you only need to call waitForUnpackLibs()?

Reply:
in the very beginning of build we run 
`runUnpackAllLibsAsync`

Then in points when we expect data to be already unpacked we `waitForUnpackLibs`
unpacking is very fast, and it's not really needed, but I want to make sure there are no issues with multitreading.


The only case where I call both is LuaInit, because of some tests which bypass `Bob.invoke()`. It is harmless to do so, because I have an extra checking if it's already unpacked.